File: Backend/NodeJS/src/config/credentials.js
import dotenv from 'dotenv';

dotenv.config();

export const dbConfig = {
    host: process.env.DB_HOST,
    user: process.env.DB_USER,
    password: process.env.DB_PASSWORD,
    database: process.env.DATABASE,
    port: process.env.DB_PORT
}

export const bucketConfig = {
    name: process.env.NAME_BUCKET,
    region: process.env.REGION_BUCKET,
    id: process.env.ID_BUCKET,
    key: process.env.KEY_BUCKET
}

File: Backend/Python/src/config/credentials.py
from dotenv import load_dotenv
import os

#Carga las variables de entorno
load_dotenv()

dbConfig = {
    'pool_name': "pool_sound_stream",
    'host': os.environ.get('DB_HOST'),
    'user': os.environ.get('DB_USER'),
    'password': os.environ.get('DB_PASSWORD'),
    'database': os.environ.get('DATABASE'),
    'port': os.environ.get('DB_PORT')
}
#Configuracion del bucket
bucketConfig = {
    'name': os.environ.get('NAME_BUCKET'),
    'region': os.environ.get('REGION_BUCKET'),
    'id': os.environ.get('ID_BUCKET'),
    'key': os.environ.get('KEY_BUCKET')
}